build.sbt

```diff
@@ -6,8 +6,7 @@ startYear     := Some(2016)
 licenses      += ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))
 
 sbtVersion in Global := "1.0.3"
-crossSbtVersions     := List("0.13.17", "1.1.4")
-scalaVersion         := "2.12.7"
```

Comment:
Any particular reason why `scalaVersion` went out?

Reply:
Usually, it would be better to use `scalaVersion` depended by sbt. If we don't have any reasons, we need not specify the `scalaVersion` in this file. (Especially this plugin is cross-built by sbt 0.13)
This is not a common sense but I haven't seen meaningful `scalaVersion` in any plugins.

Reply:
Fair point. I'm going to test and release this shortly - thanks for the clarification.